cypress/support/order-commands.js

```diff
@@ -12,7 +12,7 @@ Cypress.Commands.add("withSupervisionLevel", {prevSubject: true}, (order, overri
   cy.get('@order');
 });
 
-Cypress.Commands.add("withActiveOrderStatus", {prevSubject: true}, (order, overrides = {}) => {
```

Comment:
Is there a reason why you changed this from being fluent chaining function, rather than updating the create deputy to use prevSubject instead?